service/src/main/java/org/cbioportal/service/impl/StructuralVariantEnrichmentServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package org.cbioportal.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.cbioportal.model.AlterationEnrichment;
+import org.cbioportal.model.MolecularProfileCaseIdentifier;
+import org.cbioportal.model.StructuralVariantCountByGene;
+import org.cbioportal.service.StructuralVariantEnrichmentService;
+import org.cbioportal.service.StructuralVariantService;
+import org.cbioportal.service.exception.MolecularProfileNotFoundException;
+import org.cbioportal.service.util.AlterationEnrichmentUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class StructuralVariantEnrichmentServiceImpl implements StructuralVariantEnrichmentService {
+    @Autowired
+    private StructuralVariantService structuralVariantService;
+    @Autowired
+    private AlterationEnrichmentUtil<StructuralVariantCountByGene> alterationEnrichmentUtil;
+
+    @Override
+    public List<AlterationEnrichment> getStructuralVariantEnrichments(
+            Map<String, List<MolecularProfileCaseIdentifier>> molecularProfileCaseSets, String enrichmentType)
+            throws MolecularProfileNotFoundException {
```

Comment:
Where is `MolecularProfileNotFoundException`  thrown? That's a checked exception, so I should get a compiler error when I remove it if something in the function body throws it, but I don't.

Reply:
Looks like it is not required